packages/mui-base/src/Portal/Portal.types.ts

```diff
@@ -12,24 +12,10 @@ export interface PortalProps {
    * By default, it uses the body of the top-level document object,
    * so it's simply `document.body` most of the time.
    */
-  container?: Element | (() => Element | null) | null;
+  container?: HTMLElement | (() => HTMLElement | null) | null;
```

Comment:
What's the reason for this change?

Reply:
Inside the component we are treating it like an `HTMLElement`, see https://github.com/mui/material-ui/pull/34807#discussion_r1000463042
Either we need a new check before passing it to or we need to fix the interface.

My rational around the change was that the description talks about an HTML Element (https://github.com/mui/material-ui/blob/master/packages/mui-base/src/Portal/Portal.d.ts#L9)
But maybe it's the description that is wrong.